pkg/spec/populators.go

```diff
@@ -20,12 +20,26 @@ import (
 	"encoding/json"
 	"fmt"
 	"sort"
+	"strconv"
 
 	log "github.com/Sirupsen/logrus"
-	"github.com/pkg/errors"
 	api_v1 "k8s.io/client-go/pkg/api/v1"
+
+	"github.com/pkg/errors"
```

Comment:
can you arrange the imports similar to the comment here https://github.com/kedgeproject/kedge/pull/149#issuecomment-315051859

Reply:
There are no kedge imports here, these are in the order of stdlib first, then third party. What are you suggesting?